Assets/MixedRealityToolkit.Extensions/Webrtc/Scripts/IPeer.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using Microsoft.MixedReality.Toolkit.Extensions.Webrtc.Delegates;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Webrtc
+{
+    public interface IPeerImpl
```

Comment:
>IPeerImpl [](start = 21, length = 9)

nit: i think we should have different files for different interfaces.  Also is there a reason for the Impl ending compared to another term that may be more telling to why there are two interfaces?

Reply:
not a good one - for clearer code-reuse i stil the (reused) method defs so that both the underlying peer and the unity events wrapper could consume it. will rename and split out to something more logical.